tests/plugins/test_amp_plugin.py

```diff
@@ -84,3 +84,65 @@ def on_fit_start(self, trainer, pl_module):
 
     with pytest.raises(SystemExit):
         trainer.fit(model)
+
+
+@pytest.mark.skipif(
+    LooseVersion(torch.__version__) < LooseVersion("1.6.0"),
+    reason="Minimal PT version is set to 1.6")
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
+def test_amp_gradient_unscale(tmpdir):
+
+    class ExtendedBoringModel(BoringModel):
+
+        def on_after_backward(self):
+            norm = torch.nn.utils.clip_grad_norm_(self.parameters(), 2)
+            if not (torch.isinf(norm) or torch.isnan(norm)):
+                assert norm.item() < 15.
+
+    model = ExtendedBoringModel()
+
+    trainer = Trainer(
+        max_epochs=2,
+        default_root_dir=os.getcwd(),
+        limit_train_batches=2,
+        limit_test_batches=2,
+        limit_val_batches=2,
+        amp_backend='native',
+        distributed_backend='ddp',
```

Comment:
The backend can't be ddp as these will hang on our drone CI builds, could you swap to `ddp_spawn`? cc @awaelchli just to make sure I'm correct

```suggestion
        distributed_backend='ddp_spawn',
```